### PR TITLE
update README, update flake, fix 'since's

### PR DESCRIPTION
**Pull request by peter-mlabs**
Can you give this a quick look and make sure I'm not misunderstanding anything? Thanks!

### CODE REVIEW

**Comment by peter-mlabs** [comment]
Closes #3

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Public discussion and assistance can be found on [discord](https://discord.gg/yGkjxrYueB) @ #liqwid-libs
+
+## What is this?
+
+A helper library to compare algorithm implementations of Plutarch code.
+
+Some of the benchmarking techniques are multi-threaded.
+
+This library helps users to write benchmarks on memory units, CPU units,
+and script size, and ranks those implementations in terms of best, worst,
```
Comment (cjay) [inline README.md:11]: The script size is only in the metadata currently, it doesn't get ranked.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+The `example/Main.hs` module demostrates the basic benchmarking process,
+with the key elements being described below. Users new to the library can read 
+the examples as code, as well as execute the benchmarks with `cabal bench` to see 
+what they do.
+
+The `Plutarch.Benchmark.Sized` module exposes a types and functions 
+to assist with generating test data. The key types are:
+
+- `SSample` (for "Sized Sample), which holds sample data along with
+  metadata about the size of the input it was generated from,
```
Comment (cjay) [inline README.md:41]: inputs

File: README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+what they do.
+
+The `Plutarch.Benchmark.Sized` module exposes a types and functions 
+to assist with generating test data. The key types are:
+
+- `SSample` (for "Sized Sample), which holds sample data along with
+  metadata about the size of the input it was generated from, 
+  the size of the sample itself, coverage, and the sample itself
+- `SUniversalGen` (for "Sized Universal Generator"), which holds
+  a pair of generators: exhaustive generators for small samples sizes,
```
Comment (cjay) [inline README.md:44]: small input sizes (also next line)

**Review by cjay — CHANGES_REQUESTED** [review]
The rest looks good.

**Comment by peter-mlabs** [comment]
Thanks @cjay! I'm going to open an issue to add script size to the benchmarking, but its not a high priority right now.

**Review by cjay — APPROVED** [review]

**Pull request merged**